### ISSUE TITLE
Font size page title overview page

### ISSUE THREAD

**Issue by maxliqd** [liqd/digitalstrategie#259]
**URL:** https://digitalstrategie-dev.liqd.net/ergebinsse/
**user:** visitor
**expected behaviour:** Page title has the same font size as teaser titles
**behaviour:** Is smaller
**important screensize:**
**device & browser:** Macbook firefox 
**Comment/Question:** 

Screenshot?

![grafik](https://user-images.githubusercontent.com/94360651/153913446-919a1e7c-89dd-4896-8534-31616ab80fc8.png)

**Comment by maxliqd** [comment]
@phillimorland I took a second look. The page title seems fine, at least not bigger. The teaser title seemed too big as it is an old block in the first screenshot. Take a look at this screenshot, where you can see the page title over the new block:

<img width="696" alt="grafik" src="https://user-images.githubusercontent.com/94360651/154297249-35f6046d-af17-4aaa-a82a-45c1325394d1.png">

**Comment by phillimorland** [comment]
@maxliqd yeah we haven't touched the old blocks as we are still uncertain what should be adapted and what we should delete, i'll removed the styling for the headline and hopefully everything should be clearer when we've got final design go ahead, I thought some elements are not confirmed yet or am I wrong there? once we know we can deleted everything that's not needed

**Comment by maxliqd** [comment]
@phillimorland You're completely right. I was a little too quick with this issue, I suppose. Sorry!

**Comment by phillimorland** [comment]
@maxliqd no no, it's not clear what we've already adapted and whats not, I also get confused :p
